js/deer-render.js

```diff
@@ -36,6 +36,7 @@ async function renderChange(mutationsList) {
                     obj = JSON.parse(localStorage.getItem(id))
                 } catch (err) { }
                 if (!obj || !obj["@id"]) {
+                    id = id.replace(/^https?:/,location.protocol) // avoid mixed content
```

Comment:
since it i sonly used once, you could put this in the fetch() or when assigned up top